### ISSUE TITLE
Allow to delete hard-linked beatmaps

### ISSUE THREAD

**Issue by rust2** [ppy/osu#31059]
### Type

Game behaviour

### Bug description

When osu! runs for the first time, it asks if you want to import beatmaps from osu!stable installation.
If agreed, there's only option to delete them one by one - from lazer song selection and either osu!stable's one or its songs folder. It would be good to have second option to also completely delete hard linked files

### Version

2024.1115.3-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18083911/compressed-logs.zip)

**Comment by bdach** [comment]
I don't understand what this is asking? Why would you want to "delete hard-linked beatmaps"? They take zero additional space on your device. If you delete the maps from both clients then the files will go away. Why would you ever want to manually delete hard links?

**Comment by peppy** [comment]
There definitely seems to be some kind of misunderstanding here.

@rust2 are you just looking for a button to delete all your beatmaps?

![osu! 2024-12-11 at 08 19 42](https://github.com/user-attachments/assets/f4a0b805-be7c-4aa4-bfb6-c04a81391158)

if not, please explain what you want in plain english (not using the word "hard link").

**Comment by rust2** [comment]
I'm sorry. I must have missed something.
I import all beatmaps when setup wizard ask me. When I don't need certain beatmap anymore, I can delete it from song selection screen but the copy will remain in stable's songs folder and I need to go there and delete it from there too

**Comment by bdach** [comment]
> but the copy will remain in stable's songs folder and I need to go there and delete it from there too.

That is correct and by design. The import from stable is supposed to be a one time operation. It is not intended to keep both clients' beatmaps synchronised indefinitely from that point onwards and I'm not sure we ever want to do that.

I don't think there's anything to fix here, this can probably just be closed.

**Comment by peppy** [comment]
Yep, it is very intentional that there's no synchronisation, to avoid one client from ever causing issues with the other.
